src/data_provider/src/sysInfoLinux.cpp

```diff
@@ -184,17 +185,40 @@ int SysInfo::getCpuCores() const
 int SysInfo::getCpuMHz() const
 {
     int retVal { 0 };
-    std::map<std::string, std::string> systemInfo;
-    getSystemInfo(WM_SYS_CPU_DIR, ":", systemInfo);
-
-    const auto& it { systemInfo.find("cpu MHz") };
+    int cpuFreq { 0 };
```

Comment:
Please do not delete the old implementation, only use the new one in case it cannot be recovered from the old implementation. The reason is that these folders are not there using docker containers.